### DISCUSSION TITLE
Code_gen for hessian and sparse_hessian

### DISCUSSION THREAD

**Discussion by a-jp** [coin-or/CppAD#224] [Q&A]
Hi, is it possible to use the code_gen feature for hessian and sparse_hessian computations?
Thanks

**Comment by bradbell** [comment]
If you build the code_gen documentation, you will get doxygen output that seems to include sparse hessians.

It is also possible to generate code using CppAD itself (and this would work for your purpose); see
https://cppad.readthedocs.io/latest/jit_get_started.cpp.html#
